### PR TITLE
Fix position of action buttons in image editor on mobile

### PR DESCRIPTION
**Pull request by parliamnt101**
…e view when viewing on mobile

This is a PR to address issue #3711.

My fix is to wrap the three columns on the page in the form instead of just having the form inside the first column. Since the first column is the only place with input fields and is the only form on the page, this should have any negative side effects. This also allows me to but the submit and delete buttons in their own column below the others. This doesn't change the view on the desktop view but moves those buttons to the bottom of the page when viewed in a smaller viewport.

### CODE REVIEW

**Comment by kapito** [comment]
Looks good, however the position seems out of place when using a 'medium-sized' viewport (~1200px width):

image

File: wagtail/wagtailimages/templates/wagtailimages/images/edit.html [hunk 1]
```diff
@@ -33,66 +33,70 @@
...
-                data-focal-point-y="{{ image.focal_point_y }}"
-                data-focal-point-width="{{ image.focal_point_width }}"
-                data-focal-point-height="{{ image.focal_point_height }}">
-
-                <img {{ rendition.attrs }} data-original-width="{{ image.width }}" data-original-height="{{ image.height }}" class="show-transparency">
-
-                <div class="current-focal-point-indicator{% if not image.has_focal_point %} hidden{% endif %}"></div>
+        <form action="{% url 'wagtailimages:edit' image.id %}" method="POST" enctype="multipart/form-data" novalidate>
+            <div class="col5">
+                    {% csrf_token %}
```
Comment (thibaudcolas) [inline wagtail/wagtailimages/templates/wagtailimages/images/edit.html]: Minor – should be right after the `form` tag instead of here.

**Comment by kapito** [comment]
https://pastebin.com/kF7AUX4Z -- Tested in Firefox 55, Chrome 60 and Safari mobile (iPhone 6S). All working without issues.

File: wagtail/images/templates/wagtailimages/images/edit.html [hunk 2]
```diff
@@ -48,59 +49,64 @@
...
 
-                <img {{ rendition.attrs }} data-original-width="{{ image.width|unlocalize }}" data-original-height="{{ image.height|unlocalize }}" class="show-transparency">
+                    <img {{ rendition.attrs }} data-original-width="{{ image.width|unlocalize }}" data-original-height="{{ image.height|unlocalize }}" class="show-transparency">
 
-                <div class="current-focal-point-indicator{% if not image.has_focal_point %} hidden{% endif %}"></div>
+                    <div class="current-focal-point-indicator{% if not image.has_focal_point %} hidden{% endif %}"></div>
 
+                </div>
+                <br />
+                <button class="button button-secondary no remove-focal-point" type="button">{% trans "Remove focal area" %}</button>
```
Comment (thibaudcolas) [inline wagtail/images/templates/wagtailimages/images/edit.html]: `type="button"` was necessary so clicking this wouldn't insta-submit the form.

**Comment by thibaudcolas** [comment]
It's in 🎉. Will ship in Wagtail 2.1.

**Pull request merged**